axon_schema.go

```diff
@@ -254,6 +255,39 @@ func loadColumnTypes(conn *sqlx.DB) error {
 	return nil
 }
 
+// loadColumnTypesPGX loads the data types for all the columns in the database using a pgx connection.
+func loadColumnTypesPGX(conn *pgx.Conn) error {
```

Comment:
We really need to get rid of sqlx (or pgx), but this is fine for "getting the project done"